docs/language-usage.rst

```diff
@@ -0,0 +1,279 @@
+Using the API
+=============
+
+The `Google Natural Language`_ API can be used to reveal the
+structure and meaning of text via powerful machine
+learning models. You can use it to extract information about
+people, places, events and much more, mentioned in text documents,
+news articles or blog posts. You can use it to understand
+sentiment about your product on social media or parse intent from
+customer conversations happening in a call center or a messaging
+app. You can analyze text uploaded in your request or integrate
+with your document storage on Google Cloud Storage.
+
+.. warning::
+
+   This is a Beta release of Google Cloud Natural Language API. This
+   API is not intended for real-time usage in critical applications.
+
+.. _Google Natural Language: https://cloud.google.com/natural-language/docs/getting-started
+
+Client
+------
+
+:class:`~gcloud.language.client.Client` objects provide a
+means to configure your application. Each instance holds
+both a ``project`` and an authenticated connection to the
+Natural Language service.
+
+For an overview of authentication in ``gcloud-python``, see
+:doc:`gcloud-auth`.
+
+Assuming your environment is set up as described in that document,
+create an instance of :class:`~gcloud.language.client.Client`.
+
+  .. code-block:: python
+
+     >>> from gcloud import language
+     >>> client = language.Client()
+
+By default the ``language`` is ``'en'`` and the ``encoding`` is
+UTF-8. To over-ride these values:
+
+  .. code-block:: python
+
+     >>> client = language.Client(language='es',
+     ...                          encoding=encoding=language.Encoding.UTF16)
+
+The encoding can be one of
+:attr:`Encoding.UTF8 <gcloud.language.document.Encoding.UTF8>`,
+:attr:`Encoding.UTF16 <gcloud.language.document.Encoding.UTF16>`, or
+:attr:`Encoding.UTF32 <gcloud.language.document.Encoding.UTF32>`.
+
+Methods
+-------
+
+The Google Natural Language API has three supported methods
+
+- `analyzeEntities`_
+- `analyzeSentiment`_
+- `annotateText`_
+
+and each method uses a `Document`_ for representing text. To
+create a :class:`~gcloud.language.document.Document`,
+
+  .. code-block:: python
+
+     >>> text_content = (
+     ...     'Google, headquartered in Mountain View, unveiled the '
+     ...     'new Android phone at the Consumer Electronic Show.  '
+     ...     'Sundar Pichai said in his keynote that users love '
+     ...     'their new Android phones.')
+     >>> document = client.document_from_text(text_content)
```

Comment:
Could this pattern be `client.document.from_text()` ? Or `client.text_document(...)` and `client.html_document()` ?